examples/validate_relative_binding.py

```diff
@@ -392,17 +391,18 @@ def pred_fn(params, mol, mol_ref):
 
 
     for epoch in range(cmd_args.epochs):
-        epoch_params = serialize_handlers(ordered_handles)
-        # dataset.shuffle()
+        dataset.shuffle()
```

Comment:
Did we want to shuffle for inference (eg. property_field is None)

Reply:
In that case it seems like it shouldn't matter? Ordering in inference, with or without the labels, is inconsequential as far as I can tell

Reply:
would it be okay for the default be to not-shuffle, and add an argument to enable it?

Reply:
Works for me